Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,188 @@
+            return FindPoorest(parsedFile);
+        }
+
+        internal static Person[] FindPoorest(Person[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo(culture);
+            var persons = new Person[0];
+            var lowestAmount = decimal.MaxValue;
+
+            for (int i = 0; i <= balances.Length - 1; i++)
+            {
+                var currentPerson = balances[i];
+                var lowestAmountOfPerson = currentPerson.CurrentBalance();
+
+                if (lowestAmount > lowestAmountOfPerson)
+                {
+                    lowestAmount = lowestAmountOfPerson;
+
+                    persons = new Person[0];
+                }
+
+                if (lowestAmount >= lowestAmountOfPerson)
+                {
+                    persons = AddPerson(currentPerson, persons);
+                }
+            }
+            return persons;
+        }
+
+        public static Person[] FindPersonWithBiggestLoss(string[] peopleAndBalances)
+        {
+            var parsedFile = BalanceParser.Parser(peopleAndBalances);
+
+            return FindBiggestLoss(parsedFile);  
+        }
+
+        private static Person[] FindBiggestLoss(Person[] balances)
+        {
+            CultureInfo.CurrentCulture = new CultureInfo(culture);
+            var persons = new Person[0];
+            var biggestLoss = decimal.MinValue; 
+
+            for (int i = 0; i <= balances.Length - 1; i++)
+            {
+                var currentPerson = balances[i]; 
+
+                if (currentPerson.GetBalance().Length <= 1)
+                {
+                    return new Person[0];
+                }
+
+                 var biggestLossPerson = currentPerson.BiggestLoss(); 
+                
+                if (biggestLossPerson > biggestLoss)
+                {
+                    biggestLoss = biggestLossPerson;
+                    persons = new Person[0]; 
+                }
+
+                if (biggestLossPerson >= biggestLoss)
+                {
+                    persons = AddPerson(currentPerson, persons);
+                }
+            }
+
+            return persons;
+        }
+
+
+        private static decimal FindBiggestLossPerson(decimal[] allAmounts)
+        {
+            var biggestLossPerson = decimal.MinValue;
+            for (int j = 0; j < allAmounts.Length - 1; j++)
+            {
+                var amount1 = allAmounts[j];
+                var amount2 = allAmounts[j + 1]; 
+                var lossForCurrentPerson = amount1 - amount2;
+
+                //check if loss is greater then the current highest loss
+                if (lossForCurrentPerson > biggestLossPerson)
+                {
+                    biggestLossPerson = lossForCurrentPerson;
+                }
+            }
+
+            return biggestLossPerson;
+        }
+
+
+        //    public static string FindRichestPerson(string[] peopleAndBalances)
+        //    {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)